src/Api/Label/GithubLabelApi.php

```diff
@@ -36,11 +37,17 @@ class GithubLabelApi implements LabelApi
      */
     private $logger;
 
-    public function __construct(Labels $labelsApi, CacheInterface $cache, LoggerInterface $logger)
+    /**
+     * @var ResultPager
+     */
+    private $resultPager;
+
+    public function __construct(Labels $labelsApi, ResultPager $resultPager, CacheInterface $cache, LoggerInterface $logger)
     {
         $this->labelsApi = $labelsApi;
         $this->cache = $cache;
         $this->logger = $logger;
+        $this->resultPager = $resultPager;
```

Comment:
Could you move this up 2 lines so that its order matches the arguments? And the same for the `private $resultPager`

Reply:
Sure